openedx/adg/lms/applications/models.py

```diff
@@ -36,6 +45,18 @@ class ApplicationHub(TimeStampedModel):
         default=False, verbose_name=_('Written Application Submitted'),
     )
     submission_date = models.DateField(null=True, blank=True, verbose_name=_('Submission Date'), )
+    prerequisite_course_status = models.CharField(
```

Comment:
since now we have two types of prereqs, i.e. Program prerqs and BU prereqs, it's better to differentiate the two by renaming this variable to `program_prereq_courses_status`. Also it's not the status of a single course, but a set of courses, so we should use plural **courses** in the variable name.

Initially, there were no bu_prereq courses, so we had named the variable `is_prerequisite_courses_passed`. But now that we have bu prereqs, we should name it appropriately like I mentioned above.

Reply:
Is this the status of multiple courses? I'm not sure if multiple courses will have same statuses at a time or not.

Reply:
This actually represents courses status for one application. Basically,

- Not Started: None of the courses are started
- In Progress: Atleast one of the courses is enrolled
- Completed: All the courses are passed